src/api/v1/donate.go

```diff
@@ -55,10 +55,25 @@ func afterDonation(c echo.Context) error {
 	}
 
 	if order.Total() < 500 {
-		// if the donation is too small, save it
-		// maybe they make multiple that sum up to over 500 eventually, lets make a record of it idk
-		// just dont give em a registration token lol
+		// keep a record of orders that are too smol
 		log.Println("Donation with PayPal order ID", order.ID, "and total", order.Total(), "is too small. token would have been", token)
+		// also check if the total of all donations with the same token is more than $5.00
+		res, err := database.DB.Query("SELECT SUM(amount) AS total FROM pending_donations WHERE token=$1", token)
```

Comment:
This doesn't make sense. `token` is the unique id of the row, so there will never be multiple rows with the same token. Instead you would need to add a paypal `payer_id` column to the table. This should be accessible as something like `order.payer.payer_id` according to [here](https://developer.paypal.com/docs/api/orders/v2/#orders_get) and [here](https://developer.paypal.com/docs/api/orders/v2/#definition-payer)

Reply:
Despite the API claiming that a `payer` will be provided when getting order details, that doesn't appear to be the case. It looks like payer details are only provided when capturing a payment, hence the [capture](https://github.com/ImpactDevelopment/ImpactServer/capture) branch.